### DISCUSSION TITLE
[Content Request] Add a label next to a session whenever new content is added

### DISCUSSION THREAD

**Discussion by GrimReaper3223** [java/devjava-content#119] [New Content Ideas]
### Content Details

I would like to propose to the devs who maintain dev.java to add some stripe or label next to the learning sessions when there are new things added to the site. For example, let's say something new was added to the 'Streams' session. So, next to the 'Streams' section, it would be interesting to have an identifier 'New' or something like that, to indicate that there is something new there.
We generally need to look at every session to check if there is anything new to be studied, and this is not always feasible, as there may be subtle changes that we do not notice.

### Website Section

Learn --> Tutorials

**Comment by danthe1st** [comment]
Are you talking about new articles (either listed on `/learn` or as part of an article series where the whole series should get that tag?) or also about updated articles?

Also, do you have any suggestions on how long an article should be considered "new"? Note that there may be some time between an article being written (the timestamp of the article/`last_updated`) and when the article is actually included (no timestamp in this repository).

**Reply by GrimReaper3223** [reply]
Hello

I mean all /learn sessions.
For example, whenever a new session update is made to any article, a "New" tag should be added next to the session, indicating that there is new content that is worth reviewing (as if it were an "insight")

Other tags would also be interesting, such as modified content (some updated content, not necessarily new) and deleted content (Such as Java language specifications that were removed in new versions, which could make certain tutorials outdated).

Regarding time, I've also been thinking about this and I don't have a good idea of how long this tag should remain visible. 
My suggestion would be to analyze the website update time. If the site is updated a lot, keeping these tags for a fixed period of time would be interesting.  Maybe 15 or 10 days. This also largely depends on the number of visits made to the website. If there are few, the time could be a little longer.
But if the site is not frequently updated, only being updated when there are new things (i.e. something that takes time), So it would be interesting to schedule the removal of tags added in the last update of the site in the next update.

But if I were to say a fixed time, I would put something standard... Maybe 15 days. 7 days is interesting, but I don't know if it would have the expected effect on those who don't have that much time to access the platform.

These are just suggestions :)

Taking advantage of this discussion, I would like to propose something else as well. I'm Brazilian and I make extensive use of the website. I realize that the translation enabled by the browser doesn't always help, keeping things confusing when translated into Portuguese.Some separate translation plugins translate everything completely wrong! Source code comments are also not translated.
As this project helped me a lot in becoming a Java developer, I would really like, when I have some free time, to provide some contributions with translation into the pt_BR language. Do you think it would be a good idea? I would translate the pages little by little. Not only the reading of the pages in the /learn session, but also the comments of the codes that are not identified by the translator.
In fact, the translator translates certain key words in Java or certain terminologies that there is no need to translate, which can make it difficult for other people in my country to study.

**Comment by Just-some-one** [comment]
maybe extend [Recently Updated Articles](https://dev.java/#updates) section to show have more link that show all pages (page mean single page or maybe whole tutorial in case if they added on same day ) sorted by date something like news and event section on [Latest News & Events](https://dev.java/#news-events) section

to make it clear each link on this one https://dev.java/learn/ is considered page / group of page depend if it has single page or multi

for example [Objects, Classes, Interfaces, Packages, and Inheritance](https://dev.java/learn/oop/) is page
but [The Collections Framework](https://dev.java/learn/api/collections-framework/) is group of page

while date be the write time and maybe time of update 
which update will  **_<code>not</code>_** to be included

1) update cause of typo
2)  minor change like rewrite some parts of page that written in confused way on first time 
3) update that fix links
4) code bug

let us take example assume that Sequenced interfaced not written in the collection framework tutorial then if we :

1) add it as separate page then we add a link on the more page with the date 
2) add as section of already existing page add link on the more page also


not sure if that would help or not

and have a nice day everyone :)

**Comment by maryphelps** [comment]
That’s a really thoughtful suggestion — having a “New” or “Updated” label next to recently changed learning sessions (like the Streams section) would definitely make it easier to keep track of updates without needing to manually check everything.

It could be helpful for learners to quickly spot new material, especially when the changes are subtle. Maybe something like a small badge or even a “Last updated” timestamp could work well.

+1 to this idea — tagging the maintainers here for visibility.
